python/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # This file should list any python package dependencies.
 nose==1.3.7
+pyspark>=2.0.0
```

Comment:
graphframes have droped support for spark below version 3.4 or something

Reply:
3.4 is pretty recent - shouldn't we keep it as broad as we can?

Reply:
so users with spark 2.4 should try it to figure out that we don't support it?

Reply:
When you say we don't support <3.4, where did that happen? I thought we supported more of a range than that.

Reply:
https://github.com/graphframes/graphframes/pull/430

Reply:
Okay, sorry I forgot. Updated the file.